### ISSUE TITLE
GPI to 2150:  transient after 100 years

### ISSUE THREAD

**Issue by jonathandmoyer** [PardeeCenterDU/IFs-Issues-Tracking#261]
This is a weird one, but after 100 years or so (so, 2120 with a base year of 2019) Austria (or was it Australia?) jumps to 90% of global power.

**Comment by caleb-petry** [comment]
@collinmeisel, thinking now, this could be related to/similar to the Taiwan and zero trade issue. Was that showing up in GPI?

**Comment by collinmeisel** [comment]
@caleb-petry Right, the Taiwan 0 trade issue didn't appear to negatively affect any variables EXCEPT the FBIC Index and GPI. My guess is it's something to do with a zero denominator under a non-zero numerator.

**Comment by PardeeCenterIFs** [comment]
I don't remember ever testing a model run past 2100, I'm glad it doesn't blow up, but I'm sure it has many issues. I'll assign it the lowest priority, but will keep the issue open.
